source/ed/connectors/gtfs_parser.h

```diff
@@ -130,11 +130,11 @@ struct GtfsData {
 };
 
 //a bit of abstraction around tz time shift to be able to change from boost::date_time::timezone if we need to
-struct period_with_utc_shift {
-    period_with_utc_shift(boost::gregorian::date_period p, boost::posix_time::time_duration dur) :
+struct PeriodWithUtcShift {
+    PeriodWithUtcShift(boost::gregorian::date_period p, boost::posix_time::time_duration dur) :
         period(p), utc_shift(dur.total_seconds() / 60)
```

Comment:
it's quite a big change to do this properly, I created a ticket for this:

http://jira.canaltp.fr/browse/NAVP-339